src/components/SystemMessageList/SystemMessageList.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from 'react';
+import {SystemMessageNotification, SystemMessageNotificationTypes} from '../SystemMessageNotification/SystemMessageNotification';
+import {getSystemMessages} from '../../api';
+import {useMount} from '../../hooks/useMount';
+
+
+export interface SystemMessageList {
+  id: number;
+  type: SystemMessageNotificationTypes;
+  description: string;
+
+}
+
+// Create function for getting and processing id to local storage
+
+const SystemMessagesList  = () => {
+
+  const [systemMessageList, setSystemMessageList] = useState<SystemMessageList[]>([]);
+
+  const [count, setCount] = useState( 0);
+
+  useMount(async () => {
+      const messages = await getSystemMessages();
+      setSystemMessageList(messages);
+
+  });
+
```

Comment:
You will need a `useEffect` callback here to update the IDs stored in session storage whenever the state updates. The state is an array of strings, and session storage always stores things as a plain old string, so you'll need to convert the string array to a string. Javascript has a method that can accomplish this called `join()`. The spec for this method is [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join).

Basically it just joins all the elements of an array into a string. By default it will separate the array entries by commas, which will work just fine for our purposes so we'll leave it alone. Like so:

```
useEffect(() => {
    window.sessionStorage.setItem('dismissedMessages', dismissedMessages.join());
}, [dismissedMessages]);
```

This should run every time the `dismissedMessages` state updates, and store the new value of that state in session storage under the key 'dismissedMessages'. We can then retrieve this state if the page ever loads, and it will last until the browser session expires.